docs/decisions/eventing-and-messaging.md

```diff
@@ -0,0 +1,67 @@
+# Separating Eventing and Messaging
+
+Decision date: 6 March 2019
+
+## What
+
+Move Channel and Subscription to the apigroup `messaging.knative.dev` from
+`eventing.knative.dev`, to indicate that Channel and Subscription concepts are
+more aligned with messaging (i.e. message routing) than eventing (i.e.
+content-aware dispatch).
+
+## New API Group charters
+
+### `messaging.knative.dev`
+
+Messaging is explicitly concerned with modelling the event transmission
+topology; the route of the message through the topology is specified by the
+developer, and the topology determines which functions receieve which events.
+
+![Channel and Subscription model](channel-subscription-model.svg)
+
+Message Channels can be used for Pub/Sub messaging where producers and consumers
+are decoupled, even temporally: either side can come and go, consumers read at
+their own rate, potentially including support for replay, etc. Channels also
+provide the building blocks for Enterprise Integration Patterns, such as Routers
+and higher level patterns such as Scatter/Gather and may support partitioning,
+and stream processing over windows.
+
+### `eventing.knative.dev`
+
+Eventing is a higher-level abstraction which removes explicit management of the
+event transmission topology and replaces it with automatically-optimized routing
+based on the contents of the event. In this way, it is similar to local
+event-driven programming with the Observer pattern.
+
+![Broker and Trigger model](broker-trigger-model.svg)
```

Comment:
For ease of updating the image, can you include a comment with a link to the Google Drawing?

For all images on the page.

Reply:
Done.